pkgs/tools/misc/minipro/default.nix

```diff
@@ -7,23 +7,25 @@
 
 stdenv.mkDerivation rec {
   pname = "minipro";
-  version = "0.5";
+  version = "0.6";
 
   src = fetchFromGitLab {
     owner = "DavidGriffith";
     repo = "minipro";
     rev = version;
-    sha256 = "sha256-Hyj2LyY7W8opjigH+QLHHbDyelC0LMgGgdN+u3nNoJc=";
+    sha256 = "sha256-6i4PYzSm6lcO/Nb6n3holrNe4TtAnG2o81ZIMAIGqOE=";
   };
 
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ libusb1 ];
   makeFlags = [
+    "VERSION=${version}"
     "PREFIX=$(out)"
     "UDEV_DIR=$(out)/lib/udev"
     "COMPLETIONS_DIR=$(out)/share/bash-completion/completions"
     "PKG_CONFIG=${pkg-config}/bin/${pkg-config.targetPrefix}pkg-config"
     "CC=${stdenv.cc.targetPrefix}cc"
+    "CFLAGS=-O2"
```

Comment:
why override cflags?

Reply:
The default `CFLAGS` has no optimization  `-O0`. It looks like it is for convenience of development running a debugger and such, but not really something one would do for a production binary.

Reply:
makes sense; my only concern is that upstream enables `-Wall -Wextra -W -Wno-sign-compare -Wno-unused-parameter` so by overriding cflags we're building with a different set of error flags

Reply:
These warnings are more important for the developers (in fact, I sent some of these in a pull request upstream to minipro) to make sure that the code is clean. But for building the release binary they are not providing any benefit.